src/launcher/util/appCompatibilityWarning.ts

```diff
@@ -65,16 +65,27 @@ export const checkEngineIsSupported: AppCompatibilityChecker = (
 };
 
 const checkMinimalRequiredAppVersions: AppCompatibilityChecker = app => {
+    const minSupportedVersion = minimalRequiredAppVersions[app.name];
+    if (minSupportedVersion === null) {
+        return incompatible(
+            'This version of nRF Connect for Desktop does not support ' +
+                `this app.`,
+            `This version of nRF Connect for Desktop does not support ` +
+                `this app "${app.displayName}". Running the currently ` +
+                `installed version will not work.`
```

Comment:
C: If you add an “anymore” above, then of course also here.

Because in this text we only write about the version of nRF Connect for Desktop but not the version of the app, I think the “Running the currently installed version will not work.” is a bit misleading. Maybe simplify it to just “Running this will not work.”?